cloudinary/cloudinary_wordpress
# Update wizard

Updates the plugin wizard

## Approach

- Updates the print screen from the API keys page
- Updates the instructions on how to set the API keys
- Updates the build process


## QA notes

- Launch the wizard from the already existing installation;
- On a new install, you should still be guided to the wizard process
